### DISCUSSION TITLE
VScode on WSL cannot see new release for extension

### DISCUSSION THREAD

**Discussion by microhobby** [microsoft/vscode-discussions#1077] [Extension Development QnA] [answered]
I have a user who reports that under WSL 2 he is not getting the updates automatically. He needs to do the following:

- Go to the extension activity bar.
- Search for the extension.
- When the extension appears on the search list, then it shows the update button and starts downloading it.

This is expected, or is some kind of issue?

**Comment by isidorn** [comment, marked as answer]
Feels like you are not an extension author, and this is a community for vscode extension authors. And your kind of question is best asked on stack overflow. Thus closing this.

**Reply by microhobby** [reply]
Yes, I am an extension author. And this was actually reported by one user of my extensions. But ok, I will report on other place.
